sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -304,7 +304,9 @@ object CommandUtils extends Logging {
       columns.map(statExprs(_, conf, attributePercentiles))
 
     val namedExpressions = expressions.map(e => Alias(e, e.toString)())
-    val statsRow = new QueryExecution(sparkSession, Aggregate(Nil, namedExpressions, relation))
+    val statsRow = new QueryExecution(sparkSession, Aggregate(Nil, namedExpressions, relation),
+      shuffleCleanupMode =
+      QueryExecution.determineShuffleCleanupMode(sparkSession.sessionState.conf))
```

Comment:
This is an internally created DataFrame, and users won't be able to hold the DataFrame reference. I think we can safely enable shuffle cleanup here?